src/platform_impl/android/mod.rs

```diff
@@ -180,47 +180,117 @@ impl<T: 'static> EventLoop<T> {
                                                             force: None,
                                                         },
                                                     ),
                                                 };
+
                                                 call_event_handler!(
                                                     event_handler,
                                                     self.window_target(),
                                                     control_flow,
                                                     event
                                                 );
                                             }
-                                        }
+                                            MotionAction::Move => {
+                                                for pointer in motion_event.pointers() {
+                                                    let location = PhysicalPosition {
+                                                        x: pointer.x() as _,
+                                                        y: pointer.y() as _,
+                                                    };
+
+                                                    let event = event::Event::WindowEvent {
+                                                        window_id,
+                                                        event: event::WindowEvent::Touch(
+                                                            event::Touch {
+                                                                device_id,
+                                                                phase: event::TouchPhase::Moved,
+                                                                location,
+                                                                id: pointer.pointer_id() as u64,
+                                                                force: None,
+                                                            },
+                                                        ),
+                                                    };
+                                                    call_event_handler!(
+                                                        event_handler,
+                                                        self.window_target(),
+                                                        control_flow,
+                                                        event
+                                                    );
+                                                }
+                                            }
+                                            MotionAction::Cancel => {
+                                                for pointer in motion_event.pointers() {
+                                                    let location = PhysicalPosition {
+                                                        x: pointer.x() as _,
```

Comment:
you previously had `0.0` here for defaults - was it intential to move back to the pointer values?

Reply:
Yes it is intentional from my side, but I have to be honest I'm unsure what the right approach is when it comes to cancelled events.

Reply:
looking through the iOS implementation, the current proposal seems to match quite well